perl/p5-net-apns-persistent/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.branches      ${perl5.default_branch}
```

Comment:
You really do need to list the branches explicitly. If the value of perl5.default_branch changes in the portgroup, that would change the subports defined by this portfile without the portfile itself changing, causing the PortIndex to be incorrect and dependents to possible break.

Reply:
Is it a decent practice to just commit to whatever version `perl5` is using?

That's the version what will get launched when `/opt/local/bin/perl` is called.

Reply:
For perl5.branches? Everything else tends to list the 2-3 latest perl versions. You could list only one version if you really wanted I guess.